### ISSUE TITLE
Consider introducing MOCKSERVER_INITIALIZATION_JSON_PATH's analogue for OpenAPI YAML file

### ISSUE THREAD

**Issue by cyber-barrista** [mock-server/mockserver#1307]
**Describe the feature request**
When it comes to the docker instance of a mockserver there're handy ENVs that enable a user to point out to mapped under the container config files (properties and expectation initialiser JSON).  Would be so convenient to have the same type of ENV for OpenAPI YAML files.

**What you are trying to do**
Trying to initialise a docker instance of mockserver with an OpenAPI YAML file.

**The solution you'd like**
Introducing something like `MOCKSERVER_INITIALIZATION_OPENAPI_YAML_PATH` which points to a file which is to be parsed and used by the server at its startup.

**Describe alternatives you've considered**
Current workaround is 
+ Spin off not initialised mockserver docker container with the OpenAPI YAML file mapped under
+ Feed the YAML to the mockserver invoking `mockserver/openapi` from the outside later on

PS. If such functionality already exists but undocumented of if there're other elegant solutions I didn't think of would be glad to hear of them. Thank you!

**Comment by jamesdbloom** [comment]
I agree this makes sense I'll look to add this in the next few weeks or so for the next release

**Comment by camilok14** [comment]
Is there any news on this suggestion ?
It is surprising for this software not to include this feature.

**Comment by jamesdbloom** [comment]
Implemented in commit eb892e8f98a71b8cb0d23c1ca6cd54a3ada9e5f5.

## New Configuration Property: `MOCKSERVER_INITIALIZATION_OPENAPI_PATH`

MockServer can now initialize expectations directly from an OpenAPI v3 specification file (YAML or JSON) at startup, without needing to wrap the spec in MockServer expectation JSON.

### Usage

**Environment variable (Docker):**
```
MOCKSERVER_INITIALIZATION_OPENAPI_PATH=/config/petstore.yaml
```

**System property:**
```
-Dmockserver.initializationOpenAPIPath=/config/petstore.yaml
```

**Java API:**
```java
ConfigurationProperties.initializationOpenAPIPath("/config/petstore.yaml");
```

### Features
- Supports YAML (.yaml, .yml) and JSON (.json) OpenAPI v3 specifications
- Supports classpath and filesystem paths
- Supports glob patterns for loading multiple spec files
- Supports file watching (`watchInitializationJson=true`) to auto-reload on spec changes
- Can be used alongside `initializationJsonPath` — JSON expectations load first, then OpenAPI expectations
- MockServer generates an expectation for each operation in the spec, with example responses derived from the schema definitions

### Docker Compose example
```yaml
services:
  mockServer:
    image: mockserver/mockserver:latest
    ports:
      - 1080:1080
    environment:
      MOCKSERVER_INITIALIZATION_OPENAPI_PATH: /config/petstore.yaml
    volumes:
      - type: bind
        source: .
        target: /config
```

### Files changed (13)
- `ConfigurationProperties.java`, `Configuration.java` — new property
- `ExpectationInitializerLoader.java` — OpenAPI loading logic
- `OpenAPIParser.java` — cache eviction for file watcher support
- `HttpState.java` — initialization trigger
- `ExpectationFileWatcher.java` — watch OpenAPI files
- 8 new unit tests, 2 new integration tests
- Consumer and internal documentation updated
